internal/manifest/iso_tree.go

```diff
@@ -65,10 +57,16 @@ func NewISOTree(m *Manifest,
 }
 
 func (p *ISOTree) getOSTreeCommits() []osTreeCommit {
+	var checksum, url string
+	if p.OSTree != nil {
+		checksum = p.OSTree.Checksum
+		url = p.OSTree.URL
+
+	}
 	return []osTreeCommit{
 		{
-			checksum: p.ostree.Parent,
-			url:      p.ostree.URL,
+			checksum: checksum,
+			url:      url,
```

Comment:
Wouldn't it make more sense to return empty slice, instead of a slice with commit that contains invalid values? I know that the ISO tree is currently used mostly by the ostree-based images, but I expect it to be eventually used by other images as well. I think that in this case, the osbuild-related code (`GenSources()`) would generate `org.osbuild.ostree` source with an empty URL and  checksum.

Reply:
Fair, but still, if it is not set in the `ISOTree` struct, the returned slice should be IMO empty.

Reply:
Yup.  Done.